### DISCUSSION TITLE
Lists modal not closing

### DISCUSSION THREAD

**Discussion by Bumboobee** [community/community#50880] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When add to list modal is open, and you try to close it by click in the close icon on right top conner, nothing happen. 
The class clicked is ```SelectMenu-closeButton```

**the html**
~~~javascript
  <button class="SelectMenu-closeButton" type="button" aria-label="Close menu" data-toggle-for="details-user-list-129840012">
      <svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" class="octicon octicon-x">
          <path d="M3.72 3.72a.75.75 0 0 1 1.06 0L8 6.94l3.22-3.22a.749.749 0 0 1 1.275.326.749.749 0 0 1-.215.734L9.06 8l3.22 3.22a.749.749 0 0 1-.326 1.275.749.749 0 0 1-.734-.215L8 9.06l-3.22 3.22a.751.751 0 0 1-1.042-.018.751.751 0 0 1-.018-1.042L6.94 8 3.72 4.78a.75.75 0 0 1 0-1.06Z"></path>
      </svg>
  </button>
~~~

**the modal** 
![image](https://user-images.githubusercontent.com/94147847/227416574-c17c9f07-bd5f-429d-b051-b1d0cb8d089f.png)

> probabily is something quicky to fix, but is very important for user expirence.

**Comment by backendbrat** [comment]
Also after clicking "create list" button, it changes to "created". But, the same issue happens; modal doesn't close. Have to reload the page to get rid of modal.

![bug](https://user-images.githubusercontent.com/79698697/227714107-ed58b80e-8770-44f5-952d-88bf6d12ce1d.PNG)

**Reply by MIRELOSZKA196** [reply]
#

**Comment by Sirleid** [comment]
👋👍

**Comment by Sirleid** [comment]
👍

**Comment by diogo-ms** [comment]
Found the same problem today, both the model not closing and the modal box not closing when creating a list from it

**Comment by Jamim** [comment]
It's a shame the bug [mentioned](https://github.com/orgs/community/discussions/50880#discussioncomment-5426026) by @backendbrat hasn't been fixed yet 🤦🏼
Duplicates of this issue:
  * #51296
  * #54245
  * #55960
  * #58026
  * #61757
  * #64121
  * #64614
  * #76291
  * #76972
  * #79361
  * #81925
  * #84201
  * #102107

**Comment by Logan-Elliott** [comment]
I'm also having the same issue. 
I'm glad to see I am not the only one.
Hopefully, we get a fix soon. :wrench:

**Comment by 4Furki4** [comment]
It's been almost a year and it's just a quick fix. That's a shame.

**Comment by jennie-jd** [comment]
I still encounter the same issue today. The create list screen still freezes for both my Chrome and MS Edge browsers. If the webpage is refreshed manually, then it shows that the list has actually already been created.

![image](https://github.com/community/community/assets/52141333/a901936e-4b55-4aff-ac7e-aff52ffbf87f)

**Comment by DavidCardoso** [comment]
I also faced this same issue today on Brave.
Hope they fix this soon.

> Maybe they could use Copilot to help them solve this quickly 😜

**Reply by rifqilubis** [reply]
Nah, probably not soon. It happened since march 2023. Either they can't or ignore it for now.

**Comment by slog198** [comment]
https://www.wps.com/d?from=a

**Comment by WilmarE** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> When add to list modal is open, and you try to close it by click in the close icon on right top conner, nothing happen. 
> The class clicked is ```SelectMenu-closeButton```
> 
> 
> **the html**
> ~~~javascript
>   <button class="SelectMenu-closeButton" type="button" aria-label="Close menu" data-toggle-for="details-user-list-129840012">
>       <svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" class="octicon octicon-x">
>           <path d="M3.72 3.72a.75.75 0 0 1 1.06 0L8 6.94l3.22-3.22a.749.749 0 0 1 1.275.326.749.749 0 0 1-.215.734L9.06 8l3.22 3.22a.749.749 0 0 1-.326 1.275.749.749 0 0 1-.734-.215L8 9.06l-3.22 3.22a.751.751 0 0 1-1.042-.018.751.751 0 0 1-.018-1.042L6.94 8 3.72 4.78a.75.75 0 0 1 0-1.06Z"></path>
>       </svg>
>   </button>
> ~~~
> 
> **the modal** 
> ![image](https://user-images.githubusercontent.com/94147847/227416574-c17c9f07-bd5f-429d-b051-b1d0cb8d089f.png)
> 
> > probabily is something quicky to fix, but is very important for user expirence. 
>

**Comment by MIRELOSZKA196** [comment]
_****_

**Comment by JokerQyou** [comment]
It's been over a year and no fix or communication was ever made from any GitHub employee. I kind of think Star List feature is totally abandoned.

**Comment by sreemanthink** [comment]
responce #128533

**Comment by ytxmobile98** [comment]
As of 2024-10-23 This is not yet fixed.
